test/maps/lua_testsuite.wmf/scripting/immovables_descriptions.lua

```diff
@@ -282,6 +287,20 @@ function test_descr:test_vision_range()
    assert_equal(17, egbase:get_building_description("barbarians_tower").vision_range)
 end
 
+function test_descr:test_building_descriptions()
+   local building_descrs = egbase.building_descriptions
```

Comment:
No reason not to test them here, just change `egbase` to `wl.Descriptions()` here and above and it should be fine.